src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -457,6 +457,27 @@ class WizardImpl implements azdata.window.Wizard {
 	}
 }
 
+class ModelViewDashboardImpl implements azdata.window.ModelViewDashboard {
+	constructor(
+		private _editor: ModelViewEditorImpl
+	) {
+	}
+
+	registerTabs(handler: (view: azdata.ModelView) => Thenable<(azdata.Tab | azdata.TabGroup)[]>): void {
+		this._editor.registerContent(async (view) => {
+			const tabs = await handler(view);
```

Comment:
...so with creation of dashboard the toolbar (or placeholder for it) should also come by default right? are we planning to do that later or not doing that?

Reply:
I have added the toolbar support in my latest commit and updated the PR description to reflect that.